src/RequestHandler/inprocess/inprocesshandler.cpp

```diff
@@ -59,25 +59,16 @@ IN_PROCESS_HANDLER::OnAsyncCompletion(
     HRESULT     hrCompletionStatus
 )
 {
-    HRESULT hr;
-    if (FAILED(hrCompletionStatus))
+    // For now we are assuming we are in our own self contained box. 
+    // TODO refactor Finished and Failure sections to handle in process and out of process failure.
```

Comment:
File issues for these TODO? How important are they for the code to function?

Reply:
I should both of these TODOs.